src/main/java/org/nuxeo/onedrive/client/OneDriveFolder.java

```diff
@@ -130,9 +125,9 @@ public URLTemplate getChildrenURL() {
     public OneDriveDeltaItemIterator delta() {
         URL url;
```

Comment:
`getDeltaUrl` could be implemented in `OneDriveItem` (Items do have Delta (includes Files and Folders))